src/wkt/src/wrappers.rs

```diff
@@ -157,10 +183,23 @@ impl crate::message::Message for UInt64Value {
     fn typename() -> &'static str {
         "type.googleapis.com/google.protobuf.UInt64Value"
     }
-    fn to_map(&self) -> Result<crate::message::Map, crate::AnyError> {
-        encode_string::<Self>(self.to_string())
+
+    fn serializer() -> Box<dyn crate::message::MessageSerializer<Self>>
+    where
+        Self: Sized,
+    {
+        Box::new(UInt64ValueSerializer)
+    }
+}
+
+struct UInt64ValueSerializer;
```

Comment:
comment: This needs to stay custom, because it has custom `to_map`, `from_map` implementations.

Same goes for `Int64Value`, `BytesValue`. They also stay custom.

Reply:
Yes. No changes have been made to these.